azurerm/internal/services/hdinsight/hdinsight_hbase_cluster_resource.go

```diff
@@ -282,6 +294,13 @@ func resourceArmHDInsightHBaseClusterRead(d *schema.ResourceData, meta interface
 		d.Set("https_endpoint", httpEndpoint)
 		sshEndpoint := azure.FindHDInsightConnectivityEndpoint("SSH", props.ConnectivityEndpoints)
 		d.Set("ssh_endpoint", sshEndpoint)
+
+		monitor, err := extensionsClient.GetMonitoringStatus(ctx, resourceGroup, name)
+		if err != nil {
+			return fmt.Errorf("Error reading monitor configuation for HDInsight Hadoop Cluster %q (Resource Group %q): %+v", name, resourceGroup, err)
```

Comment:
Cosmetic change, we're removing the duplication of the word "Error" from error messages since these are prefixed with `Error:` anyway.
```suggestion
			return fmt.Errorf("failed reading monitor configuration for HDInsight Hadoop Cluster %q (Resource Group %q): %+v", name, resourceGroup, err)
```